style.css

```diff
@@ -0,0 +1,889 @@
+    font-size: 10px;
+  }
+
+  .icon-detail {
+    display: flex;
+    justify-content: center;
+  }
+
+  .descrip {
+    position: absolute;
+    display: flex;
+    text-align: center;
+    width: 90%;
+    left: 5%;
+    right: 5%;
+    top: 120px;
+  }
+
+  .nonebnt {
+    display: none;
+  }
+
+  .z {
+    margin-left: 15%;
+  }
+
+  .a {
+    margin-right: 15%;
+  }
+
+  .bntsumitJoin {
+    display: none;
+  }
+
+  .parcipateBnt {
+    display: none;
+  }
+
+  .LinksumitJoin {
+    display: flex;
+    background-color: #272a31;
+    justify-content: center;
+    justify-items: center;
+    padding-bottom: 30px;
+  }
+
+  .text1 {
+    color: #000;
+    width: 250px;
+    height: auto;
+  }
+
+  .text2 {
+    color: chocolate;
+    width: 250px;
+    height: auto;
+  }
+
+  .text3 {
+    color: #272a31;
+    width: 250px;
+    height: auto;
+  }
+
+  .speakers {
+    display: grid;
+    grid-template-columns: 200px 200px;
+    grid-template-rows: repeat(3, 120px);
+    row-gap: 0;
+    column-gap: 150px;
+    margin-left: 20%;
+    justify-content: left;
+    margin-top: 0;
+    width: 100%;
+  }
+
+  .title-Main {
+    position: absolute;
+    top: 5px;
+    left: 45%;
+    color: hsl(223, 16%, 91%);
+    font-family: 'Lato', sans-serif;
+    line-height: 1.5;
+    font-size: large;
+  }
+
+  .title-Main2 {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    color: hsl(222, 11%, 17%);
+    font-family: 'Lato', sans-serif;
+    line-height: 1.5;
+    font-size: large;
+  }
+
+  .speaker-person1 {
+    display: flex;
+    line-height: 1.5;
+    font-size: 14px;
+  }
+
+  .speaker-person2 {
+    display: flex;
+    line-height: 1.5;
+    font-size: 14px;
+  }
+
+  .speaker-person3 {
+    display: flex;
+    line-height: 1.5;
+    font-size: 14px;
+  }
+
+  .speaker-person4 {
+    display: flex;
+    line-height: 1.5;
+    font-size: 14px;
+  }
+
+  .speaker-person5 {
+    display: flex;
+    line-height: 1.5;
+    font-size: 14px;
+  }
+
+  .speaker-person6 {
+    display: flex;
+    line-height: 1.5;
+    font-size: 14px;
+  }
+
+  .imgP1 {
+    width: 100px;
+    height: 100px;
+    padding-right: 10px;
+  }
+
+  .imgP2 {
+    width: 100px;
+    height: 100px;
+    padding-right: 10px;
+  }
+
+  .imgP3 {
+    width: 100px;
+    height: 100px;
+    padding-right: 10px;
+  }
+
+  .imgP4 {
+    width: 100px;
+    height: 100px;
+    padding-right: 10px;
+  }
+
+  .imgP5 {
+    width: 100px;
+    height: 100px;
+    padding-right: 10px;
+  }
+
+  .imgP6 {
+    width: 100px;
+    height: 100px;
+    padding-right: 10px;
+  }
+
+  .footer {
+    position: relative;
+    height: 60vh;
+    width: auto;
+  }
+
+  .footer1 {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    height: 30vh;
+    width: 100%;
+    background-color: #272a31;
+  }
+
+  .footer1 .footer11 {
+    display: flex;
+    margin-top: 20px;
+    justify-content: center;
+    color: #d3d3d3;
+    width: 100%;
+  }
+
```

Comment:
As suggested by the previous reviewer, kindly use the given color for the list of partners in this section, and also decrease the space between them to match the design guideline. 🥺 

| Your Design | Figma Design |
| :---:   | :---: | 
| <img width="1470" alt="image" src="https://user-images.githubusercontent.com/73854999/227220233-98046f46-d6bf-4a8e-8e26-dfd10203bdab.png">|<img width="1036" alt="image" src="https://user-images.githubusercontent.com/73854999/227220422-03779dfa-603d-4935-841f-710849677129.png">|